db/migrate/20160606040848_create_payments.rb

```diff
@@ -0,0 +1,18 @@
+class CreatePayments < ActiveRecord::Migration
+  def change
+    create_table :payments do |t|
+      t.string :first_name
+      t.string :last_name
+      t.string :last4
+      t.decimal :amount, precision: 12, scale: 3
+      t.string :authorization_code
+      t.integer :status, default: 0
```

Comment:
I would also add `null: false` for the other fields: first name, last name, last4, amount, status.
Not for authorization code, since that one only gets set whe n the payment goes through, i guess?